### ISSUE TITLE
With the `<amp-carousel>` 0.2 script, the right arrow sometimes shows on the last slide

### ISSUE THREAD

**Issue by kienstra** [ampproject/amphtml#25510]
## What's the issue?

In an `<amp-carousel>` using the [0.2 script](https://amp.dev/documentation/components/amp-carousel/?format=websites):
```html
<script async custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.2.js"></script>
```
...sometimes a right arrow appears even when there's only 1 slide, or if the last slide is showing. This issue doesn't appear with the [0.1 script](https://amp.dev/documentation/components/amp-carousel-v0.1/?format=websites).

Maybe this isn't an issue with AMP HTML, and there's some styling we could apply to prevent this. I first saw this in the AMP Plugin for WordPress: https://github.com/ampproject/amp-wp/pull/3115

## How do we reproduce the issue?

Here's a [CodePen of it](https://codepen.io/ryankienstra/pen/bGGMJaJ). On changing `amp-carousel-0.2.js` back to `amp-carousel-0.1.js` in that CodePen, the issue doesn't appear anymore.

## Steps to reproduce in WordPress

1. Checkout [this PR's branch](https://github.com/ampproject/amp-wp/pull/3115), and activate Twenty Nineteen
2. Create a new post
3. Add a Gallery block 
4. Add only one image to it, [like this](https://picsum.photos/id/96/640/400)
5. Select 'Display as carousel'
6. Click 'Preview,' and go to the AMP URL
7. Expected: Because there's only 1 image in the carousel, there shouldn't be a right arrow
8. Actual: There's a right arrow, but clicking it doesn't do anything:

<img width="1041" alt="carousel-here" src="https://user-images.githubusercontent.com/4063887/68443863-ca5e7200-019a-11ea-9738-7d851d95086c.png">

9. Create another Gallery block, but with at least 2 images
10. Select 'Display as carousel'
11. Click 'Preview,' and go to the AMP URL
12. Expected: When the last slide in the carousel displays, the right arrow shouldn't be visible
13. Actual: It's still visible:
<img width="941" alt="last-slide" src="https://user-images.githubusercontent.com/4063887/68444021-435dc980-019b-11ea-9568-3ace725464fb.png">

This isn't a common issue, and I didn't see it with most Core themes. Maybe there's some styling that Twenty Nineteen has that's associated with this issue.
## What browsers are affected?

Chrome Desktop Version 78.0.3904.87 (Official Build) (64-bit)

This issue doesn't seem to exist on Firefox or Safari

## Which AMP version is affected?

1911062056110

I first noticed this issue yesterday.

**Comment by westonruter** [comment]
@kienstra I'm not seeing the same thing. If I open that [CodePen](https://codepen.io/ryankienstra/pen/bGGMJaJ), for the first carousel, I see no arrow:

> ![image](https://user-images.githubusercontent.com/134745/68507302-6ed6c780-0220-11ea-891b-5c2f9f3cc296.png)

And in the second I see no next arrow:

> ![image](https://user-images.githubusercontent.com/134745/68507349-88780f00-0220-11ea-8d9a-f8b9d9ffcdc3.png)

**Comment by kienstra** [comment]
>@kienstra I'm not seeing the same thing. If I open that CodePen, for the first carousel, I see no arrow:

OK, thanks for testing this. Maybe it's my version of Chrome.

**Comment by kienstra** [comment]
Since you can't reproduce this with that [CodePen](https://codepen.io/ryankienstra/pen/bGGMJaJ) or [on WordPress](https://github.com/ampproject/amp-wp/pull/3115#issuecomment-551973012), maybe this issue is unique to my browser. I'm closing this.

**Comment by schlessera** [comment]
I can see the same issue as @kienstra in the Codepen he posted. However, it seems to randomly appear and disappear based on resize/repaints...

![Screen Recording 2019-11-08 at 11 30 PM](https://user-images.githubusercontent.com/83631/68515101-22bb6f80-0280-11ea-927f-4e96705481cb.gif)

This issue should be reopened.

**Comment by kienstra** [comment]
@schlessera, thanks for testing this issue and posting the `.gif` with details.
